{{cookiecutter.project_slug}}/home/migrations/0002_load_initial_data.py

```diff
@@ -21,7 +22,6 @@ def create_homepage(apps, schema_editor):
 
 
 def create_site(apps, schema_editor):
-    Site = apps.get_model("sites", "Site")
```

Comment:
This needs to stay

Reply:
Import at the top, this one doesn't work, tested on staging.

Reply:
It should work now that it depends on a sites migration. See the docs here: https://docs.djangoproject.com/en/3.0/topics/migrations/#accessing-models-from-other-apps